divi/src/BlockMemoryPoolTransactionCollector.h

```diff
@@ -14,7 +14,7 @@
 
 #include <boost/tuple/tuple.hpp>
 #include <boost/thread/recursive_mutex.hpp>
-
+#include <main.h>
 
```

Comment:
We're going to remove this one eventually, I hope?

Reply:
is removed